wavespectra/input/ndbc.py

```diff
@@ -48,22 +50,29 @@ def read_ndbc(url, directional=True, dd=10.0, chunks={}):
 
     """
     dset = xr.open_dataset(url).chunk(chunks)
-    return from_ndbc(dset, directional=directional, dd=dd)
+    return from_ndbc(dset, directional=directional, dd=dd, weight_coeff=weight_coeff)
 
 
-def _construct_spectra(ef, swd1, swd2, swr1, swr2, dir):
+def _construct_spectra(ef, swd1, swd2, swr1, swr2, dir, weight_coeff: bool = False):
```

Comment:
Question (not that I am reviewing or anything. but if the integral parameters remain, and this boolean solves the negative values, wouldn't it be better if this `weight_coeff` Defaults to True?

Reply:
Well, I guess @rafa-guedes needs to decide, but, just hear me out, thinking out loud here.

I've been thinking and I cannot think about any scenario where people (anyone) would actually _want_ to have negative energy density values. I imagine that the vast majority of people would appreciate that the quickes/simplest reconstruction of NDBC 2D Spectra is correct from a physical point (ie, all energy >= 0).  Which is why I thought about defaulting to True.

But maybe I am missing the big picture or some obvious case where someone would want the raw reconstruction ( IMO the negative values are just a mathematical artifact, hence not desired)